src/modals/Rule/EditRule.vue

```diff
@@ -277,11 +277,23 @@ import { Rule } from '../../entities/index.js'
 						input-label="Source ID *" />
 
 					<NcTextField
-						label="File Path"
+						label="File path"
 						required
 						:value.sync="ruleItem.configuration.fetch_file.filePath"
 						placeholder="path.to.fetch.file" />
 
+					<NcTextField
+						label="File path in sub object(s) (optional)"
+						required
+						:value.sync="ruleItem.configuration.fetch_file.subObjectFilepath"
+						placeholder="path.to.fetch.file.objects" />
+
+					<NcTextField
+						label="Object id path (optional)"
+						required
+						:value.sync="ruleItem.configuration.fetch_file.objectIdPath"
+						placeholder="path.to.fetch.file.objects" />
+
```

Comment:
You are marking these fields as `required` eventhough you are displaying in the text that they are optional.